pkg/util/util.go

```diff
@@ -5,12 +5,50 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"os"
+	"text/template"
+
+	f "github.com/flyteorg/flytectl/pkg/filesystemutils"
```

Comment:
Instead of package name util, can we create config_util instead with function related to config creation there

Reply:
no-sandbox also use these function